doc/man7/openssl-env.pod

```diff
@@ -47,9 +47,10 @@ See L<openssl-provider(1)>.
 =item B<OPENSSL_WIN32_UTF8>
 
 If set, then L<UI_OpenSSL(3)> returns UTF-8 encoded strings, rather than
-ones encoded in the current locale, and
+ones encoded in the current code page, and
 the L<openssl(1)> program also transcodes the command-line parameters
 from wide characters to UTF-8.
+This behavior is only done on Microsoft Windows platforms.
```

Comment:
This environment variable is only useful on windows.

Reply:
I tweaked the wording, but I think "useful" is the wrong sense.  Didn't you try to make passwords usefully utf8, or example? :)

Reply:
Yeah, but that has nothing to do with it.  *This variable is only relevant on Windows*, and you can clearly see that from the `GetEnvironmentVariableW` call.

Reply:
I was commenting on "only useful on windows."

Yes, this variable is only checked on Windows.  Which is what it now says.